### ISSUE TITLE
Remove default comments in new scripts

### ISSUE THREAD

**Issue by Skorpio-X** [godotengine/godot#5711]
The default comments which are added to new scripts automatically are maybe useful one or two times for beginners, but then become an annoyance because you have to delete them manually every time you create a new script. Just remove them completely. The documentation should explain how everything works. I mean these comments:

```
# member variables here, example:
# var a=2
# var b="textvar"

# Called every time the node is added to the scene.
# Initialization here
```

**Comment by Zylann** [comment]
Maybe have the ability to set a different template?

**Comment by green-coder** [comment]
Hi. Do we want the user to have the possibility to modify the template or do we want to let him specify a different one? (I would like to try to provide a PR for this issue for the Hacktoberfest).

**Comment by Hinsbart** [comment]
@green-coder I would prefer the specify template approach, as proposed in #4132.
I also already started working on that a few days ago ^^

**Comment by green-coder** [comment]
Ok, I will leave it to you then.

**Comment by nunodonato** [comment]
@Hinsbart are you still on this?

**Comment by reduz** [comment]
I mean, would not remove them

**Comment by reduz** [comment]
I would remove them. It is very important that they are useful the first
times. instead would offer the option to use a different template, if not
present already

On Jun 2, 2017 12:06 PM, "Nuno Donato" <notifications@github.com> wrote:

@Hinsbart <https://github.com/hinsbart> are you still on this?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/godotengine/godot/issues/5711#issuecomment-305815090>,
or mute the thread

**Comment by toger5** [comment]
I think this is best solved with the template approach, too.

**Comment by Hinsbart** [comment]
Closing as fixed by #9158. You can add a custom template without the comments now.
Still, we might want to add some default templates (like one without comments).
Feel free to open a new issue about that.
